build/azure-pipelines/sdl-scan.yml

```diff
@@ -142,11 +143,15 @@ stages:
               GITHUB_TOKEN: "$(github-distro-mixin-password)"
             displayName: Download Symbols
 
+          - powershell: ls "$(agent.builddirectory)\\scanbin\\VSCode-win32-$(VSCODE_ARCH)\\pdb\\*.pdb"
```

Comment:
The gulp task can instead verify the presence of pdbs and add necessary debug logging